parlai/core/torch_agent.py

```diff
@@ -745,31 +897,30 @@ def _check_truncate(self, vec, truncate, truncate_left=False):
         else:
             return vec[:truncate]
 
-    def _set_text_vec(self, obs, truncate, split_lines):
+    def _make_long_tensor(self, vec):
```

Comment:
We can just use calls to `torch.LongTensor()` directly. If it's already a long tensor, it just returns itself.